exchange/exchange.go

```diff
@@ -575,15 +575,18 @@ func applyDealSupport(bidRequest *openrtb2.BidRequest, auc *auction, bidCategory
 	for impID, topBidsPerImp := range auc.winningBidsByBidder {
 		impDeal := impDealMap[impID]
 		for bidder, topBidsPerBidder := range topBidsPerImp {
-			maxBid := bidsToUpdate(multiBid, bidder.String())
-
+			bidderNormalized, bidderFound := openrtb_ext.NormalizeBidderName(bidder.String())
+			if !bidderFound {
+				continue
```

Comment:
This line isn't covered by tests, do you think you can add a test where the bidder is not found from the `NormalizeBidderName` call?

Reply:
Done. Also refactored `TestApplyDealSupport` a little bit.